plugins/xlsx-worbook-sink/src/index.ts

```diff
@@ -0,0 +1,18 @@
+import { FlatfileListener } from "@flatfile/listener";
+
+import { PluginOptions, run } from "./plugin";
+
+/**
+ * Excel Workbook export plugin for Flatfile.
+ *
+ * @param opts - plugin config options
+ */
+export const xlsxSinkPlugin = (opts: PluginOptions = {}) => {
+  return (listener: FlatfileListener) => {
+    listener.filter({ job: "workbook:downloadExcelWorkbook" }, () => {
```

Comment:
For now, job name will be hardcoded but we can add a config param later if necessary.

Reply:
It was designed to export ALL sheets in a Flatfile workbook into an excel-like workbook therefore, probably only on a workbook.